src/store.js

```diff
@@ -41,16 +55,22 @@ const store = new Vuex.Store({
     zipcodesReceived(state, zipcodes) {
       state.zipcodes = zipcodes;
     },
-    filtersReceived(state, filters) {
-      state.filters = filters;
-    },
     viewToggled(state) {
       state.view = state.view === 'map' ? 'list' : 'map';
+    },
+    setFilters(state, filters) {
+      state.filters = {...state.filters, ...filters}
+    },
+  },
+  getters: {
```

Comment:
I wonder if store getters are memoized in the same way that computed props are in components....that would be cool.